panels/history/ha-panel-history.html

```diff
@@ -120,9 +120,13 @@
     _currentDate: {
       type: String,
       value: function () {
-        var value = new Date();
-        value.setHours(0, 0, 0);
-        return value.toISOString().split('T')[0];
+        var today = new Date();
```

Comment:
I, for one, welcome our new Date object overlord. I have added a commit with this as an alternative.